src/diffusers/pipelines/ltx/pipeline_ltx_i2v_long_multi_prompt.py

```diff
@@ -0,0 +1,1256 @@
+        if cond_image is not None and cond_strength > 0:
+            img = self.video_processor.preprocess(cond_image, height=height, width=width)
+            img = img.to(device=device, dtype=self.vae.dtype)
+            enc = self.vae.encode(img.unsqueeze(2))  # [B, C, 1, h, w]
+            init_latents = enc.latent_dist.mode() if hasattr(enc, "latent_dist") else enc.latents
+            init_latents = init_latents.to(torch.float32)
+            # Normalize the encoded latents (standard VAE pipeline operation)
+            init_latents = LTXPipeline._normalize_latents(
+                init_latents, self.vae.latents_mean, self.vae.latents_std, self.vae.config.scaling_factor
+            )
+            if negative_index_latents is None:
+                negative_index_latents = init_latents
+            # Blend only first latent frame
+            latents[:, :, 0, :, :] = init_latents.squeeze(2)
+
+        # 4. Prepare sigma schedule; honor explicit sigmas; disable shifting here (handled by scheduler)
+        sigmas_reset = None
+        n_steps_reset = None
+        if sigmas is not None:
+            s = torch.tensor(sigmas, dtype=torch.float32) if not isinstance(sigmas, torch.Tensor) else sigmas
+            # ComfyUI parity: do not filter steps here by threshold; skip inside the loop before scheduler.step
+            sigmas_reset = s
+            self.scheduler.set_timesteps(sigmas=s, device=device)
+        else:
+            if num_inference_steps is None:
+                num_inference_steps = 50
+            n_steps_reset = num_inference_steps
+            self.scheduler.set_timesteps(num_inference_steps=num_inference_steps, device=device)
+
+        timesteps = self.scheduler.timesteps
+        self._num_timesteps = len(timesteps)
+
+        # 5. Sliding windows in latent frames
+        tile_size_lat = max(1, temporal_tile_size // self.vae_temporal_compression_ratio)
+        overlap_lat = max(0, temporal_overlap // self.vae_temporal_compression_ratio)
+        windows = self._split_into_temporal_windows(
+            latent_num_frames, tile_size_lat, overlap_lat, self.vae_temporal_compression_ratio
+        )
+
+        # 6. Multi-prompt segments parsing
+        segment_texts = self._parse_prompt_segments(prompt, prompt_segments)
+
+        out_latents = None
+        first_window_latents = None
+
+        # 7. Process each temporal window
+        for w_idx, (w_start, w_end) in enumerate(windows):
+            if self.interrupt:
+                break
+
+            # 7.1 Encode prompt embeddings per window segment
+            seg_index = min(w_idx, len(segment_texts) - 1) if segment_texts else 0
+            pos_text = segment_texts[seg_index] if segment_texts else (prompt if isinstance(prompt, str) else "")
+            (
+                prompt_embeds,
+                prompt_attention_mask,
+                negative_prompt_embeds,
+                negative_prompt_attention_mask,
+            ) = LTXPipeline.encode_prompt(
```

Comment:
```suggestion
            ) = self.encode_prompt(
```

we can copy over the method and add a `#Copied from` statement